db/schema.rb

```diff
@@ -126,6 +126,8 @@
     t.text "terms_and_conditions"
     t.integer "board_layout", default: 0, null: false
     t.boolean "registration_enabled", default: true, null: false
+    t.boolean "ask_for_location", default: false, null: false
```

Comment:
Without some helper text on the admin panel it might be very unclear what this is referring to, maybe `request_team_location` or at least adding helper text explaining what this option does would be good.